### PR TITLE
docs: enable non-root user to run pouch commands without sudo

### PR DESCRIPTION
**Pull request by Ace-Tang**
Signed-off-by: Ace-Tang <aceapril@126.com>

### CODE REVIEW

File: INSTALLATION.md [hunk 1]
```diff
@@ -126,6 +126,32 @@ sudo systemctl start pouch
 
 Afterwards, you can pull an image and run PouchContainer containers.
 
+## Run pouch command with non-root users
+
+To enable users other than root can run pouch command directly without `sudo`, pouchd creates a unix socket owned by pouch group. Users just need have pouch group on the machine and add user to pouch group.
+
+**1. create the pouch group**
+
+If you want to unix socket to be owned by pouch group, make sure it exist or create it.
```
Comment (Letty5411) [inline INSTALLATION.md]: How about:
If you want the unix socket to be owned by pouch group, make sure it exists or you need to create it.

File: INSTALLATION.md [hunk 2]
```diff
@@ -126,6 +126,32 @@ sudo systemctl start pouch
 
 Afterwards, you can pull an image and run PouchContainer containers.
 
+## Run pouch command with non-root users
+
+To enable users other than root can run pouch command directly without `sudo`, pouchd creates a unix socket owned by pouch group. Users just need have pouch group on the machine and add user to pouch group.
```
Comment (Letty5411) [inline INSTALLATION.md]: need have -> need to have

**Comment by Ace-Tang** [comment]
Good advice, updated, PTAL, @Letty5411

File: README.md [hunk 3]
```diff
@@ -66,7 +67,7 @@ PouchContainer offers a more "application centric" approach for application deve
 
 ## Getting Started
 
-You can easily setup a basic PouchContainer environment, see [INSTALLATION.md](INSTALLATION.md). You'll need to install a few packages before starting `pouchd`, which starts a container management service. The service can be accessed through the `pouch` CLI or RPC calls. For more details, please refer to [CLI Manual](docs/commandline) and [API Manual](docs/api).
+You can easily setup a basic PouchContainer environment, see [INSTALLATION.md](INSTALLATION.md). You'll need to install a few packages before starting `pouchd`, which starts a container management service. The service can be accessed through the `pouch` CLI or RPC calls. For more details, please refer to [CLI Manual](docs/commandline), [API Manual](docs/api)and [USER Manual](docs/user).
```
Comment (zhuangqh) [inline README.md]: miss a space here.
```
 ...api)and [USER Manual](docs/user)
       ↑
```

**Comment by Ace-Tang** [comment]
PTAL, @Letty5411 @zhuangqh

**Comment by zhuangqh** [comment]
LGTM

**Pull request merged**